test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs

```diff
@@ -100,21 +110,22 @@ public static IEnumerable<object[]> MessageSizesData
         {
             get
             {
-                yield return new object[] { new string('A', 5 * 1024)};
-                yield return new object[] { new string('A', 5 * 1024 * 1024 + 32)};
+                yield return new object[] { new string('A', 5 * 1024), new WebSocketsTransport()};
+                yield return new object[] { new string('A', 5 * 1024 * 1024 + 32), new WebSocketsTransport() };
+                yield return new object[] { new string('A', 5 * 1024), new LongPollingTransport(new HttpClient())};
+                yield return new object[] { new string('A', 5 * 1024 * 1024 + 32), new LongPollingTransport(new HttpClient())};
             }
         }
 
         [ConditionalTheory]
         [OSSkipCondition(OperatingSystems.Windows, WindowsVersions.Win7, WindowsVersions.Win2008R2, SkipReason = "No WebSockets Client for this platform")]
         [MemberData(nameof(MessageSizesData))]
-        public async Task ConnectionCanSendAndReceiveDifferentMessageSizesWebSocketsTransport(string message)
+        public async Task ConnectionCanSendAndReceiveDifferentMessageSizesWebSocketsTransport(string message, ITransport transport)
```

Comment:
This test (as indicated by its name) is used to test the logic we have in the WebSockets transport to combine messages which don't fit the buffer/have endOfMessage flag set to false. As such this test is specific to WebSocketsTransport and doesn't have to be run against other transports. We already have a test above that test that sending and receiving data using different transports works.